ubyssey/static/src/js/components/GallerySlide.jsx

```diff
@@ -3,6 +3,10 @@ var GallerySlide = React.createClass({
         var slideStyle = { width: this.props.width };
         var imageStyle = { backgroundImage: "url('" + this.props.src + "')" };
 
+        if (!this.props.caption) {
+            $('.slide-credit').css('padding-top', '20px')
```

Comment:
Generally we try to avoid using jQuery to manipulate elements inside React components. When it comes to styles, React actually has built-in support for props- and state-driven style rules like so:

```jsx
var style = this.props.caption ? {paddingTop: 20px} : null;
<p className="slide-credit" style={style} />
```

However, in this case I don't think we need to be mutating the element style using JavaScript. This is due to bad code that I wrote, but a more elegant solution would be to wrap both the caption and credit elements in a single container with `padding-top: 20px` defined in the stylesheet.

```jsx
<div className='slide-meta'>
  { this.props.caption ? caption : null}
  { this.props.credit ? credit : null}
</div>
```

```css 
.slide-meta {
  padding-top: 20px;
}
```